src/Services/Air/AirParser.js

```diff
@@ -531,6 +520,55 @@ const airGetTicket = function (obj) {
   return getTicketFromEtr.call(this, etr, obj);
 };
 
+function airGetTicketsErrorHandler(rsp) {
+  let errorInfo;
+  let code;
+  try {
+    errorInfo = (
+      rsp.detail[`common_${this.uapi_version}:ErrorInfo`]
+      || rsp.detail['air:AvailabilityErrorInfo']
+    );
+    code = errorInfo[`common_${this.uapi_version}:Code`];
+  } catch (err) {
+    throw new RequestRuntimeError.UnhandledError(null, new AirRuntimeError(rsp));
+  }
+  // General Air Service error
+  if (code === '3000') {
+    if (
+      rsp.faultcode !== undefined
+      && rsp.faultstring !== undefined
+      && /has no tickets/.test(rsp.faultstring) === true
```

Comment:
A regexp test will return boolean `true` in case if it's passed and `false` in case if it's not, I don't think it's necessary to make a check if the value is true or not.